docs/pages/guides/environment-variables.md

```diff
@@ -85,7 +83,7 @@ A dotenv file is a file with all environment variables, and their value, within
 
 #### From **app.config.js**
 
-Below you can see the dynamic manifest using this `dotenv` library. It imports the `/config` module to automatically load the `.env` and merge it with `process.env`. You can also use it without merging it to `process.env`, [read more about that here](https://github.com/motdotla/dotenv#config).
+Below you can see the dynamic app config using this `dotenv` library. It imports the `/config` module to automatically load the `.env` and merge it with `process.env`. You can also use it without merging it to `process.env`, [read more about that here](https://github.com/motdotla/dotenv#config).
```

Comment:
```suggestion
Below you can see the dynamic app config using this `dotenv` library. It imports the `/config` module to automatically load the `.env` and merge it with `process.env`. You can also use it without merging it to `process.env`. [See more information about `dotenv` config](https://github.com/motdotla/dotenv#config).
```

(Unrelated to this PR) I don't like the word using "here" every time our docs redirect to a link. So I left small suggestion to update it.